phase1/master.cfg

```
@@ -562,7 +562,7 @@ c["schedulers"].append(
 
 
 def IsNoMasterBuild(step):
-    return step.getProperty("branch") != "main"
+    return step.getProperty("branch") != "main" and step.getProperty("branch") != "master"
```

```suggestion
    branch = step.getProperty("branch")
    return branch not in ['main', 'master']
```

nitpick, looks more idiomatic/readable?

---

Sure please go ahead

---

You're signing commits, feel free to merge it yourself so the signature is kept.